.buildkite/pipeline_pr.py

```diff
@@ -48,8 +48,10 @@
         "./tools/devtool -y make_release",
     )
 
-if not changed_files or any(
-    x.suffix in [".rs", ".toml", ".lock"] for x in changed_files
+if (
+    not pipeline.args.no_kani
+    and not changed_files
+    or any(x.suffix in [".rs", ".toml", ".lock"] for x in changed_files)
```

Comment:
This needs to be
```suggestion
if (
    not pipeline.args.no_kani
    and (not changed_files
    or any(x.suffix in [".rs", ".toml", ".lock"] for x in changed_files))
```

as `and` takes precedence over `or`. As you've written it, the --no-kani flag will be ignored if a .rs/.toml/.lock file is touched.

Reply:
good catch, fixed